sdk/tests/js/webgl-test-utils.js

```diff
@@ -3296,6 +3296,95 @@ function linearChannelToSRGB(value) {
     return Math.trunc(value * 255 + 0.5);
 }
 
+/**
+ * Return the named color in the specified color space.
+ * @param {string} colorName The name of the color to convert.
+ *        Supported color names are:
+ *            'Red', which is maximum sRGB red.
+ *            'Green', which is maximum sRGB green.
+ * @param {string} colorSpace The color space to convert to. Supported
+          color spaces are:
+ *            null, which is treated as sRGB
+ *            'srgb'
+ *            'display-p3'.
+ * @return {!Array.<number>} color The color in the specified color
+ *        space as an 8-bit RGBA array with unpremultiplied alpha.
+ */
+var namedColorInColorSpace = function(colorName, colorSpace) {
+  var result;
+  switch (colorSpace) {
+    case null:
```

Comment:
If `null` is changed to `undefined` in `unpackColorSpacesToTest`, above, then this would have to be changed too.

Reply:
done